src/lua-function-list.json

```diff
@@ -306,13 +316,14 @@
     "startStopWatch": "startStopWatch( watchID )",
     "stopSounds": "stopSounds()",
     "stopStopWatch": "stopStopWatch( watchID )",
-    "suffix": "suffix(text)",
+    "suffix": "suffix(text, [writingFunction], [foregroundColor], [backgroundColor], [windowName])",
     "tempAlias": "aliasID = tempAlias(regex, code to do)",
+    "tempAnsiColorTrigger": "tempAnsiColorTrigger(foregroundColor, backgroundColor, code, expireAfter)",
```

Comment:
Would it be reasonable to change `code` to `ANSI color` to differentiate it from `tempColorTrigger` ?

Reply:
The way to change it here is to change it in the wiki